index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+<!-- The Links --> 
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="font-awesome/css/font-awesome.css"> 
+</head>
+<body>
+<!-- The Header -->   
+  <header class="header"> 
+  <div class="contect-info" >
+    <span class="time" >12:00</span>  
+    <span class="fa fa-battery-full"></span> 
+    <span class="fa fa-wifi"></span>
+    <span class="fa fa-signal"></span>
+  </div>
+  <div >  
+      <span class="fa fa-bars" id="open"></span>  
+      <h1 class="logo">Gulalai Company</h1> 
+         
+  </div>
+
+  </header>
+  <section class="explain container">
+    <h1>Hey There.
+      I’m Gulalai</h1> 
+    <h3>I’am a Software Developer</h3>
+    <h5> I can help you build a product , 
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</h5> 
```

Comment:
- [ ] The subtitle should follow the hierarchy of the heading tags. So an `h2` tag would be semantically correct and the screen readers and other technologies will thank you. Please change that.
 - [ ] There is a typo in the template subtitle and you accidentally copied that. Please double-check the content and make sure that there are no typos in your portfolio.
 - [ ] The paragraph below the subtitle is not a  heading and would be better wrapped with the `p` tag. Kindly change that to make it semantically correct.